codec-dns/src/test/java/io/netty5/handler/codec/dns/DefaultDnsRecordDecoderTest.java

```diff
@@ -104,7 +107,7 @@ public void testDecompressCompressPointer() {
             buffer.readerOffset(0).writerOffset(10);
             assertEquals(buffer, uncompressed);
         } finally {
-            Resource.dispose(uncompressed);
```

Comment:
@chrisvest maybe as a follow up but I wonder if we can  have some sort of "check style" rule for that ?

Reply:
I'll make a note for looking into that.